orchestra/static/orchestra/todos/todo-list.html

```diff
@@ -15,35 +15,95 @@ <h3>
     <div class="row section-body" ng-if="todoList.ready">
       <div class="col-lg-12 col-md-12 col-sm-12">
         <form class="new-todo">
-          <select name="todoList" id="todoList" ng-model="todoList.newTodoTaskId">
-            <option value="" selected>Select owner</option>
-            <option value="{{task.id}}" ng-repeat="task in todoList.possibleTasks">{{todoList.steps[task.step_slug].name}}</option>
-          </select>
-          <input class="new-todo__description"
-                 type="text"
-                 ng-model="todoList.newTodoDescription"
-                 placeholder="Description">
-          <button type="submit"
-             class="btn btn-primary btn-sm edit-save-handle"
-             ng-disabled="!todoList.canAddTodo()"
-             ng-click="todoList.addTodo()">
-            Add
-          </button>
+          <div class="new-todo__box">
+            <select name="todoList" id="todoList" ng-model="todoList.newTodoTaskId">
+              <option value="" selected>Select owner</option>
+              <option value="{{task.id}}" ng-repeat="task in todoList.possibleTasks">{{todoList.steps[task.step_slug].name}}</option>
+            </select>
+          </div>
+          <div class="new-todo__box new-todo__box-details">
+            <input class="new-todo__description"
+                   type="text"
+                   ng-model="todoList.newTodoDescription"
+                   placeholder="Description">
+            <div class="pull-right">
+              <div class="new-todo__description__datetime">
+                <label>Start</label>
```

Comment:
for new and existing todos, i prefer "Start by" and "Due on"

disagree?

Reply:
I ask for Meredith's thought on the presentation. "Start" and "Due" use less space without losing the intended meaning.